fix building rootfs in docker issue

akgnah opened the pull request.

StephenInVamrs reviewed (commented):
Please remove space on line 70, 75, 80, 84, 86 of file README.md

akgnah committed:
fix building rootfs in docker issue

To use Docker build rootfs, you need to provide --privileged option to run,
live-build will use debootstrap, which needs to mount some directories and
needs real root permissions.
